marklogic-data-hub-central/ui/src/components/queries/managing/manage-query-modal/manage-query.tsx

```diff
@@ -129,9 +128,23 @@ const QueryModal = (props) => {
                 query = selectedQuery;
             }
         })
+
+        //hadle array datatype 
```

Comment:
Not sure what this is for. can you pls explain.

Reply:
Removed.

Reply:
Comments are good to have and I was asking for the intent of the code after this comment.